LoopKitUI/Views/Information Screens/InformationView.swift

```diff
@@ -9,7 +9,7 @@
 import SwiftUI
 
 public enum PresentationMode {
-    case modal, flow
```

Comment:
Seems `PresentationMode` is now shared, so perhaps it has earned its own file?

Reply:
Hmm, this is such a small enum that I'm not sure it's worth it, but I'm happy to do whatever you think would be best

Reply:
New files are pretty cheap, and it seems weird that this lives in `Information Screens/InformationView.swift` now that it applies to all of the settings screens, no?

Reply:
Changed!